rosbag2_performance/rosbag2_benchmarking/plotting/data_examples/example-data-10Hz-4Mb-0cache.txt

```diff
@@ -0,0 +1,33 @@
+[INFO] [1591705625.572989661] [rosbag2_transport]: Subscribed to topic '/image'
```

Comment:
what's the purpose of these cache files? Also I think "[Callback]" is actually not part of the output.

Reply:
These cache files are parts of our preliminary study. Actually the good idea is to remove them as there are no really longer a part of these packages.